src/amo/components/AddonAdminLinks/index.js

```diff
@@ -48,10 +48,12 @@ export class AddonAdminLinksBase extends React.Component<InternalProps> {
       return null;
     }
 
-    const showCodeReviewLink = hasCodeReviewPermission && !isTheme(addon.type);
-    const showThemeReviewLink = hasThemeReviewPermission && isTheme(addon.type);
+    const showCodeReviewLink =
+      hasCodeReviewPermission && ADDON_TYPE_STATIC_THEME !== addon.type;
+    const showThemeReviewLink =
+      hasThemeReviewPermission && ADDON_TYPE_STATIC_THEME === addon.type;
     const showContentReviewLink =
-      hasContentReviewPermission && !isTheme(addon.type);
+      hasContentReviewPermission && ADDON_TYPE_STATIC_THEME !== addon.type;
```

Comment:
How about introducing a `isTheme` variable above and use it?

```js
const isTheme = addon.type === ADDON_TYPE_STATIC_THEME;

const showThemeReviewLink = hasThemeReviewPermission && isTheme;
// ...
```

Reply:
good idea :), fixed